GVRf/Framework/framework/src/main/java/org/gearvrf/GVRViewManager.java

```diff
@@ -694,14 +695,22 @@ protected void captureCenterEye() {
         if (mScreenshotCenterCallback == null) {
             return;
         }
+        GVRRenderTexture eyeCaptureRenderTexture = mRenderBundle.getEyeCaptureRenderTexture();
```

Comment:
Getting (and creating) this render texture needs to be done only if multiview is in use. Please move to the if's for begin and end.